library/include/modules/Units.h

```diff
@@ -205,5 +205,13 @@ DFHACK_EXPORT extern const std::vector<int32_t> stress_cutoffs;
 DFHACK_EXPORT int getStressCategory(df::unit *unit);
 DFHACK_EXPORT int getStressCategoryRaw(int32_t stress_level);
 
+enum ActionTypeGroup {All, Movement, MovementFeet, Offensive, Work};
+DFHACK_EXPORT void subtractActionTimer(df::unit *unit, int amount, int affectedActionType);
+DFHACK_EXPORT void subtractActionTimerCategory(df::unit *unit, int amount, int affectedActionTypes);
+DFHACK_EXPORT void multiplyActionTimer(df::unit *unit, float amount, int affectedActionType);
+DFHACK_EXPORT void multiplyActionTimerCategory(df::unit *unit, float amount, int affectedActionTypes);
+DFHACK_EXPORT void setActionTimer(df::unit *unit, int amount, int affectedActionType);
+DFHACK_EXPORT void setActionTimerCategory(df::unit *unit, int amount, int affectedActionTypes);
```

Comment:
The *Category calls should take the third parameter as type `ActionTypeGroup` to indicate that it's not just any int.
Also, the non-*Category signatures should probably take a `df::unit_action_type` neh?